package.nix

```diff
@@ -213,6 +201,7 @@ in {
     autoconf-archive
     autoreconfHook
     pkg-config
+    doxygen
```

Comment:
Is this for the dev shell? I think we can do in another way fwiw in the `flake.nix`, where I combine the deps of a few derivations already

Reply:
Yes, but indeed we may want to rework this later.